flutter_twilio_conversations_web/lib/interop/classes/channel.dart

```diff
@@ -21,7 +22,7 @@ class TwilioConversationsChannel extends EventEmitter {
   external String? uniqueName;
 
   external dynamic getMessages(int? pageSize, int? anchor, String? direction);
-  external dynamic prepareMessage();
+  external MessageBuilder prepareMessage();
```

Comment:
Defined this so that JS know which methods to call.  In message_method the only change is to change `dynamic` to `MessageBuilder` and the rest of the interface remained the same